fdbserver/DataDistribution.actor.cpp

```diff
@@ -3217,16 +3216,15 @@ ACTOR Future<Void> teamTracker(DDTeamCollection* self, Reference<TCTeamInfo> tea
 						rs.priority = maxPriority;
 
 						self->output.send(rs);
-						if (deterministicRandom()->random01() < 0.01) {
-							TraceEvent("SendRelocateToDDQx100", self->distributorId)
-							    .detail("Primary", self->primary)
-							    .detail("Team", team->getDesc())
-							    .detail("KeyBegin", rs.keys.begin)
-							    .detail("KeyEnd", rs.keys.end)
-							    .detail("Priority", rs.priority)
-							    .detail("TeamFailedMachines", team->size() - serversLeft)
-							    .detail("TeamOKMachines", serversLeft);
-						}
+						TraceEvent("SendRelocateToDDQx100", self->distributorId)
+						    .suppressFor(1.0)
```

Comment:
This will change the semantics of the trace events -- it's no longer sampling at 1%. So either reverting the change or changing the type of trace event is needed.

Reply:
the old behavior is not ideal and we should replace the behavior with suppressFor() for all other places. 

The trace event name is changed.